node/src/BaseClient.ts

```diff
@@ -3494,6 +3495,36 @@ export class BaseClient {
         return this.createWritePromise(createObjectRefcount(key));
     }
 
+    /**
+     * Copies the value stored at the `source` to the `destination` key if the `destination` key does not
+     * yet exist.
+     *
+     * See https://valkey.io/commands/copy/ for more details.
+     *
+     * since Valkey version 6.2.0.
```

Comment:
I think we usually put this after the `@returns` or `Command Response` section, can you please move it there?